tools/launchdev.sh

```diff
@@ -18,8 +18,8 @@ while getopts ":r:gsc" o; do
         g)
             use_gunicorn=true
             ;;
-        s)
-            skip_examples=true
+        x)
+            copy_examples=true
```

Comment:
What was the problem, `-g` flag collided with `s` or something else?

Just curious because I tested `-g` and it worked for sure.